vespa-feed-client-cli/src/main/java/ai/vespa/feed/client/CliClient.java

```diff
@@ -75,6 +79,20 @@ private static FeedClient createFeedClient(CliArguments cliArgs) throws CliArgum
         return builder.build();
     }
 
+    private static JsonStreamFeeder createJsonFeeder(CliArguments cliArgs) throws CliArguments.CliArgumentsException, IOException {
+        FeedClient feedClient = createFeedClient(cliArgs);
+        JsonStreamFeeder.Builder builder = JsonStreamFeeder.builder(feedClient);
+        cliArgs.timeout().ifPresent(builder::withTimeout);
+        cliArgs.route().ifPresent(builder::withRoute);
+        cliArgs.traceLevel().ifPresent(builder::withTracelevel);
+        return builder.build();
+    }
+
+    private InputStream createFeedInputStream(CliArguments cliArgs) throws CliArguments.CliArgumentsException, IOException {
+        return new BufferedInputStream(
```

Comment:
Buffering already done in the json class, so no need for that here, it will only be overhead.